WIP-NEW-DOCS.md

```diff
@@ -0,0 +1,74 @@
+# Dev docs
+
+## Common ns aliases
+
+```clojure
+(:require
+  [clojuratica.core/->wl! :as wl])
+```
+
+## Open questions
+
+* What is the `parse` ns about? Turn `com.wolfram.jlink.Expr` to Clojure data? Thus likely
+  the opposite of `clojuratica.core/->wl!` ?
+* Q: What does `wl/wl` do, vs. `wl/eval` ? A: Eval is alias to wl, an evaluator instance.
+* Why do we want to support `:parse/custom-parse-symbols` (and not simply support either a sym or a set of symbols as the dispatch-val?) Also, why do we force the user to set the `:custom-parse` flag? Is always 
+ invoking a multimethod considered too expensive?
```

Comment:
To me, using defmulti as an extension point for users is something quite common, while requiring an extra flag to actually enable it is not. Personally, I try to avoid multimethods until unavoidable, but in this case it seems to be the perfect tool. So I'd vote for not requiring the flag.

Reply:
One thing that suprises me about the flag approach is that the core defines `custom-parse for `Entity` and `EntityProperty`, but due to the flag requirement, they are unused. But if you suddenly want to add you own custom parsing, and thus set the flag, not only will it impact your stuff, but you will activate also these two. Isn't it weird? If they should be "on demand" then I'd rather have them in a separate ns, which you may or may not require (as next.jdbc does with its extended date-time support etc.)